FlowCrypt/build.gradle

```diff
@@ -404,10 +404,10 @@ dependencies {
     implementation "androidx.cardview:cardview:${rootProject.ext.androidxBaseVersion}"
     implementation 'androidx.browser:browser:1.3.0'
     implementation "androidx.recyclerview:recyclerview:${rootProject.ext.recyclerViewVersion}"
-    implementation "androidx.recyclerview:recyclerview-selection:${rootProject.ext.recyclerViewVersion}"
+    implementation "androidx.recyclerview:recyclerview-selection:1.1.0"
     implementation 'androidx.constraintlayout:constraintlayout:2.1.0'
     implementation "androidx.test.espresso:espresso-idling-resource:${rootProject.ext.espressoVersion}"
-    implementation "androidx.lifecycle:lifecycle-extensions:${rootProject.ext.lifecycleVersion}"
+    implementation "androidx.lifecycle:lifecycle-extensions:2.2.0"
```

Comment:
I have a question: why versions of some libraries are defined via named constants, and some specified directly? And why in this case it has changed from constant to direct version number specification?

Reply:
Early Google released a library and sub-libraries with the same version number. But recently Google has started to release the main library without changing sub-libraries. That's why I had to use a predefined number version instead of a constant